src/Tools/babylon.assetsManager.ts

```diff
@@ -669,6 +669,118 @@ module BABYLON {
         }
     }
 
+    /**
+     * Define a task used by {BABYLON.AssetsManager} to load atlas texture
+     */
+    export class AtlasTextureAssetTask extends AbstractAssetTask {

```

Comment:
We should rename it to JSONAssetTask so it could be used for several other formats

Reply:
I will rename it.